case-studies/policy/usecase.md

```diff
@@ -0,0 +1,67 @@
+# Agent --> API-Server
+
+### Flow apply rule from API server
+
+``````mermaid
+sequenceDiagram
+autonumber
+actor U as User
+participant APIS as API Server
+participant Agent as Agent
+participant DB as Database
+U ->> APIS: apply rule
+APIS ->> APIS: gen hash
+APIS ->> DB: save rule and hash to database
+APIS ->> U: Response
+Agent ->> APIS: call interval check rule has been hashed
+APIS ->> APIS: compare hashed rule request with hash in db
+APIS ->> Agent: response rule + hash if there are changes
+Agent ->> Agent: appy new rule and save hash from response
+``````
+
+### Flow change rule from Agent
+
+``````mermaid
+sequenceDiagram
+autonumber
+participant APIS as API Server
+participant Agent as Agent
+participant DB as Database
+Agent ->> Agent: interval check rule local
```

Comment:
interval compare local ruler with hash code. If there is changed, agent ask api server give the ruler.

Reply:
basically, we will NOT allow change rule from agent nodes.